tests/acceptance/features/coreApiSharePublicLink1/createPublicLinkShare.feature

```diff
@@ -59,10 +59,11 @@ Feature: create a public link share
       | uid_owner              | %username%      |
       | name                   |                 |
     And the public should be able to download the last publicly shared file using the new public WebDAV API with password "%public%" and the content should be "Random data"
```

Comment:
This step hould not be necessary, right?
```suggestion
```

Reply:
I think it is. In the scenario public link is created with password. So public download has three conditions:
- to be able to download with password
- download fail with wrong password
- download fail with no password

Reply:
Does that Then step save response?

Reply:
no it does not, now